### PR TITLE
[tests-only][full-ci]add tests setting ANTIVIRUS_MAX_SCAN_SIZE env

### PR DESCRIPTION
**Pull request by S-Panta**
## Description
This PR adds API test for file upload with different upload limit threshold. The added scenarios are
- upload a file with virus smaller than the upload threshold
- upload a file with virus larger than the upload threshold

## Related Issue

- Part of <https://github.com/owncloud/ocis/issues/6255>

## How Has This Been Tested?

- test environment:
- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 1]
```diff
@@ -127,3 +127,34 @@ Feature: antivirus
...
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicarcom2.zip" to "/aFileWithVirus.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And as "Alice" file "/aFileWithVirus.txt" should exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:160]: Not needed
```suggestion
```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 2]
```diff
@@ -127,3 +127,34 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+  @env-config
+  Scenario Outline: upload a file with virus smaller than the upload threshold
+    Given the config "ANTIVIRUS_MAX_SCAN_SIZE" has been set to "100"
+    Given using <dav-path-version> DAV path
```
Comment (grgprarup) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:134]:
```suggestion
    And using <dav-path-version> DAV path
```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 3]
```diff
@@ -127,3 +127,34 @@ Feature: antivirus
...
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+    
+  @env-config
+  Scenario Outline: upload a file with virus larger than the upload threshold
+    Given the config "ANTIVIRUS_MAX_SCAN_SIZE" has been set to "100"
+    Given using <dav-path-version> DAV path
```
Comment (grgprarup) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:150]:
```suggestion
    And using <dav-path-version> DAV path
```

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 4]
```diff
@@ -14,11 +14,11 @@ Feature: antivirus
...
     And the content of file "/normalfile.txt" for user "Alice" should be:
-    """
-    This is a testfile.
+      """
+      This is a testfile.
 
-    Cheers.
-    """
+      Cheers.
+      """
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:21]: don't think these changes are needed

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 5]
```diff
@@ -55,11 +55,11 @@ Feature: antivirus
...
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:62]: don't think these changes are needed

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 6]
```diff
@@ -127,3 +127,32 @@ Feature: antivirus
...
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+  @env-config
+  Scenario Outline: upload a file with virus larger than the upload threshold
+    Given the config "ANTIVIRUS_MAX_SCAN_SIZE" has been set to "100"
+    And using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicarcom2.zip" to "/aFileWithVirus.txt" using the WebDAV API
```
Comment (SwikritiT) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:151]: Maybe this should also be a `zip` file
```suggestion
    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicarcom2.zip" to "/aFileWithVirus.zip" using the WebDAV API
```

**Review by saw-jan — CHANGES_REQUESTED** [review]
Is this new file required? I didn't see it being used
`tests/acceptance/filesForUpload/filesWithVirus/eicarcom2/eicar_com.zip`

File: tests/acceptance/features/apiAntivirus/antivirus.feature [hunk 7]
```diff
@@ -212,3 +212,31 @@ Feature: antivirus
...
+      | old              |
+      | new              |
+      | spaces           |
+
+  @env-config
+  Scenario Outline: upload a file with virus larger than the upload threshold
+    Given the config "ANTIVIRUS_MAX_SCAN_SIZE" has been set to "100"
+    And using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicarcom.zip" to "/aFileWithVirus.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
```
Comment (SagarGi) [inline tests/acceptance/features/apiAntivirus/antivirus.feature:236]: Any notification that use gets after upload?

Comment (S-Panta) [reply tests/acceptance/features/apiAntivirus/antivirus.feature:236]: If the virus file size is greater than the threshold, it doesn't scan the file and thus no notification is shown just for the virus upload.

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by SwikritiT — APPROVED** [review]
LGTM

**Review by saw-jan — APPROVED** [review]

**Pull request merged**